scalafmt-tests/src/test/scala/org/scalafmt/util/HasTests.scala

```diff
@@ -90,15 +96,10 @@ trait HasTests extends FormatAssertions {
               |$c""".stripMargin
           )
       }
-    val style: ScalafmtConfig = {
-      val pathPatternToReplace =
-        if (OsSpecific.isWindows) s"""\\\\.*""" else "/.*"
```

Comment:
is this because `BuildInfo` uses `/` regardless of OS?

Reply:
I refactored the code to use `nio.file.Path.relativize(..).getName(0)` to avoid dealing with regular expressions and OS-specific file separators.